cpu/atxmega/Makefile.include

```diff
@@ -0,0 +1,38 @@
+export CPU_ATXMEGA 1
+
+# CPU ROM/RAM
+ifneq (,$(findstring atxmega8,$(CPU_MODEL)))
+	RAM_LEN = 1K
+	ROM_LEN = 8K
```

Comment:
Style (here and below)
```suggestion
  RAM_LEN = 1K
  ROM_LEN = 8K
```
(you are using tabs, and 2 spaces should be used for consistency with other Makefiles)